Docker/entrypoint.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/sh
 cd /var/www/html/mpmanager
-php composer.phar install
+#download composer
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "if (hash_file('sha384', 'composer-setup.php') === 'dac665fdc30fdd8ec78b38b9800061b4150413ff2e3b6f88543c636f7cd84f6db9189d43a81e5503cda447da73c7e5b6') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php
+ls -la
+php -r "unlink('composer-setup.php');"
+mv composer.phar /usr/local/bin/composer
+
+composer install
+
```

Comment:
Could you please move this entire section into the Docker build file itself? I didn't realise we are running `composer install` from the entrypoint script. It should be part of the immutable image rather than the runtime container.

Reply:
Its tricky to move the entrypoint to the dockerfile. As far as I know, the entry point is triggered right after the container creation. As the current state mounts the volume while the container creation `composer install` should be done when the files are present. 

We could potentially move Line:3 to 10 into the docker-file but having them here makes contextually more sense

Reply:
Alright, I see... we need this to be done on the runtime side to get hot-reloads in the dev server.

So let's leave it like it is then. However, could you make it more clear that the `entrypoint` script is specific to the dev container? For example by re-naming it to `start_dev_container` or something like that.